tests/Unit/Models/MarketModelTest.php

```diff
@@ -2,19 +2,24 @@
 
 namespace Tests\Unit\Models;
 
-use Tests\TestCase;
-use Illuminate\Foundation\Testing\DatabaseMigrations;
 use App\Market;
+use App\Sponsor;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
 
 class MarketModelTest extends TestCase
 {
     use DatabaseMigrations;
 
     protected $market;
+    protected $sponsor;
     protected function setUp()
     {
         parent::setUp();
         $this->market = factory(Market::class)->create();
+        $this->sponsor = factory(Sponsor::class)->create([
+            'id' => $this->market->sponsor_id = 10,
```

Comment:
You should be able to get the sponsor. `$this->sponsor = $this->market->sponsor;`